tests/cpp/operator/test_cast_gated_swiglu.cu

```diff
@@ -100,21 +108,28 @@ void performTest(const size_t rows, const size_t cols) {
   compareResults("output_c", output_c, ref_output_c.get(), true, atol, rtol);
 }
 
-std::vector<std::pair<size_t, size_t>> test_cases = {
+std::vector<std::vector<size_t>> test_cases = {
   {128, 128},
   {256, 256},
   {768, 1024},
-  // {256, 65536},
-  // {2048, 12288},
-  // {65536, 128},
-  // {16384, 6144},
+  {256, 65536},
+  {2048, 12288},
+  {65536, 128},
+  {65536, 160},
+  {16384, 1616},
+  {1, 128},
+  {1, 1296},
+  {1, 16},
+  {5, 160},
+  {5, 4, 3, 160},
+  {217, 256},
 };
```

Comment:
Being a bit more judicious with the tests:
```suggestion
std::vector<std::vector<size_t>> test_cases = {
  {128, 128},
  {256, 256},
  {768, 1024},
  {256, 65536},
  {2048, 12288},
  {65536, 128},
  {217, 256},
  {1296},
  {5, 4, 3, 160},
};
```

Reply:
:+1: